### PR TITLE
feature/global-oda chart

### PR DESCRIPTION
**Pull request by aleesteele**

### CODE REVIEW

File: forced-migration/global-oda/js/area.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  $('#hcContainer').highcharts({
+    data: {
+      googleSpreadsheetKey: '1P-JsP2LcKwqkTzKtyqmpQGkALbLg3Q59hiI_xZ9jeXw',
+      googleSpreadsheetWorksheet: 1
+    },
+    chart: {
+      zoomType: 'x',
+      type: 'area'
+    },
+    ['#4D6E79', '#303D43', '#907561', '#781F30', '#EC382A', '#61884D', '#C9AC4D', '#52496D', '#5AA992', '#887295', '#2576CE', '#aa266a'],
```
Comment (jnschrag) [inline forced-migration/global-oda/js/area.js]: This is missing the `colors:` property so the chart doesn't work. Always be sure to check to make sure your chart works when submitting it for review.

**Review by jnschrag — CHANGES_REQUESTED** [review]
- Let's remove the jQuery dependency and just run this on vanilla JS

File: forced-migration/global-oda/js/area.js (hunk 1 above)
Comment (aleesteele) [reply forced-migration/global-oda/js/area.js]: changed + pushed.

**Comment by aleesteele** [comment]
- Rounded x-axis to whole numbers
- Corrected color
- Removed jquery
- Refactored for vanilla JS

**Comment by aleesteele** [comment]
Saw these outstanding pull requests, and figured I'd fix 'em up!

**Review by jnschrag — APPROVED** [review]

**Pull request merged**